### ISSUE TITLE
Cannot submit commercial url for oSC19

### ISSUE THREAD

**Issue by ishwon** [openSUSE/osem#2521]
Submitting SlideDeck & Vimeo urls to oSC 2019 page is returning the following message:

```
The change you wanted was rejected.
Maybe you tried to change something you didn't have access to.
```

**Steps to reproduce**

1. Go to https://events.opensuse.org
2. Click on "My Proposals"
3. Click on "Edit"
4. Click on "Commercials"
5. Paste URL and click on the "Create commercial" button.

Tried with both http & https urls.

**Comment by Ana06** [comment]
Related to (or a duplicate of) https://github.com/openSUSE/osem/issues/2409

**Comment by hennevogel** [comment]
Fixed in #2522, please confirm @ishwon
